### PR TITLE
Fix sub_selector used for view_thread

### PR DESCRIPTION
**Pull request by cancan101**
Closes https://github.com/KartikTalwar/gmail.js/issues/731

### CODE REVIEW

File: src/gmail.js [hunk 1]
```diff
@@ -280,7 +280,9 @@ var Gmail = function(localJQuery) {
     };
 
     api.check.is_thread = function() {
+        // this should match the sub_selector (nH -> if)
         var check_1 = $(".nH .if").children(":eq(1)").children().children(":eq(1)").children();
+        // Bu ->
```
Comment (josteink) [inline src/gmail.js:285]: We probably don't want these comments here to be part of this PR.

**Review by josteink — CHANGES_REQUESTED** [review]

File: src/gmail.js (hunk 1 above)
Comment (cancan101) [reply src/gmail.js:285]: These comments were intentional as they relate back to how the selector/sub selector values are used in multiple places in the code. I had to spend some time understanding the coupling.

Comment (cancan101) [reply src/gmail.js:285]: Yea happy to make this more clear. Basically the `Bu` was one of the selectors and `nH` is the other. There are two "code" paths to flag this as a thread based on the selector.

Comment (cancan101) [reply src/gmail.js:285]: I can add something along the lines of this comment ^ if you think that will make it clear

Comment (josteink) [reply src/gmail.js]: This looks good now.

**Comment by josteink** [comment]
Code change looks OK to me.

As for testing, I can't reproduce the original issue, but I can confirm that this code still seems to work for me none the less, so it shouldn't have caused any regressions at least.

Consider it merged!

**Review by josteink — APPROVED** [review]

**Pull request merged**